backend/src/user/user.controller.ts

```diff
@@ -0,0 +1,13 @@
+import {UserService} from "./user.service";
+import {Controller, Post} from "@nestjs/common";
+
+@Controller('createuser')
```

Comment:
polecam poczytać o konwencji nzewnictwa w REST API - jeśli definiujemy jakąś podścieżke dla kontrolera to tak żeby wszystkie powiązane z nim endpointy logicznie rozwijały tą scieżke. Przy tym nazewnictwie ten kontroler można użyć tylko do tworzenia usera. Najlepszą nazwą tutaj byłoby po prostu "users"